exporter/otlpexporter/cfgMeta.yaml

```diff
@@ -0,0 +1,162 @@
+type: '*otlpexporter.Config'
+fields:
+- name: timeout
```

Comment:
Unless order is necessary (I think for doc purposes would probably be sorted alphabetically) it's usually easier to work with names as maps, e.g.:

```yaml
fields:
  timeout:
    type: time.Duration
```

Thoughts?

Reply:
So, I changed over to the map, but after looking at the generated yaml files for a few days I decided I preferred the list version because the map version means that any key you're looking at may be a user-generated field name or a member of the `field` struct. To find out, you have to recall what's in the field struct or look at the context of the key, which always ended up being a little bit of mental work. The list version seems a little simpler/cleaner to me because instead of being a graph of two types, map and field, it's just a graph of one type. Also, I think it's useful to keep the ordering because it matches the order of the declarations and the yaml and config can be compared easily. Can change to map though if folks feel otherwise!